src/qualimap/qualimap_rnaseq/config.vsh.yaml

```diff
@@ -0,0 +1,111 @@
+    last_sha: 54721c6946daf6d602d7069dc127deef9cbe6b33
+description: |
+  Qualimap RNA-seq QC RNA-seq QC reports quality control metrics and bias estimations 
+  which are specific for whole transcriptome sequencing, including reads genomic 
+  origin, junction analysis, transcript coverage and 5’-3’ bias computation.
+
+argument_groups:
+- name: "Input"
+  arguments: 
+  - name: "--input"
+    type: file
+    required: true
+    example: alignment.bam
+    description: Path to the sequence alignment file in BAM format, produced by a splicing-aware aligner.
+  - name: "--gtf"
+    type: file
+    required: true
+    example: annotations.gtf
+    description: Path to genomic annotations in Ensembl GTF format.
+
+- name: "Output"
+  arguments: 
+  - name: "--output"
+    direction: output
+    type: file
+    required: true
+    example: rnaseq_qc_results.txt
+    description: Text file containing the RNAseq QC results.
+  - name: "--output_counts"
+    type: file
+    required: false
+    direction: output
+    description: Output file for computed counts.
+  - name: "--output_report"
+    type: file
+    direction: output
+    required: false
+    example: report.html
+    description: Report output file. Supported formats are PDF or HTML.
+
+- name: "Optional"
+  arguments: 
+  - name: "--pr_bases"
+    type: integer
+    required: false
+    default: 100
+    min: 1
+    description: Number of upstream/downstream nucleotide bases to compute 5'-3' bias (default = 100).
+  - name: "--tr_bias"
+    type: integer
+    required: false
+    default: 1000
+    min: 1
+    description: Number of top highly expressed transcripts to compute 5'-3' bias (default = 1000).
+  - name: "--algorithm"
+    type: string
+    required: false
+    choices: ["uniquely-mapped-reads", "proportional"]
+    default: uniquely-mapped-reads
+    description: Counting algorithm (uniquely-mapped-reads (default) or proportional).
+  - name: "--sequencing_protocol"
+    type: string
+    required: false
+    choices: ["non-strand-specific", "strand-specific-reverse", "strand-specific-forward"]
+    default: non-strand-specific
+    description: Sequencing library protocol (strand-specific-forward, strand-specific-reverse or non-strand-specific (default)).
+  - name: "--paired"
+    type: boolean_true
+    description: Setting this flag for paired-end experiments will result in counting fragments instead of reads.
+  - name: "--sorted"
+    type: boolean_true
+    description: Setting this flag indicates that the input file is already sorted by name. If flag is not set, additional sorting by name will be performed. Only requiredfor paired-end analysis.
+  - name: "--java_memory_size"
+    type: string
+    required: false
+    default: 4G 
+    description: maximum Java heap memory size, default = 4G.
+
+resources:
+  - type: bash_script
+    path: script.sh
+test_resources:
+  - type: bash_script
+    path: test.sh
+  - path: test_data/
+
+engines:
+- type: docker
```

Comment:
Don't forget to add the version number to /var/software_versions.txt

Reply:
hardcoded for now, cli tool doesn't seem to have param to fetch version